tests/test_zero_shot_prompt.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from prompt_templates.zero_shot.zero_shot_prompt import ZeroShotPrompt
+
+
+class TextTemplateTests(unittest.TestCase):
```

Comment:
Nice one! But why not a test class for each prompt template we define. So in this case the test class name would be ZeroShotPromptTest ...

Reply:
sorry, that was the initial name of the class, but I forgot to change it afterwards